Src/GoogleApis/Apis/Services/BaseClientService.cs

```diff
@@ -36,8 +36,10 @@
 namespace Google.Apis.Services
 {
     /// <summary>
-    /// A thread-safe base class for a client service which provides common mechanism for all services, like 
-    /// serialization and GZip support.
+    /// A base class for a client service which provides common mechanism for all services, like 
+    /// serialization and GZip support.  Although this class isn't thread-safe (it contains a reference to
+    /// <see cref="HttpClient"/> which isn't thread-safe itself), it's recommended to create a single service for
+    /// several server calls.

```

Comment:
I could see this being a bit confusing. Maybe simplify to describe the intent:

It should be safe to use a service instance to make server requests concurrently from multiple threads.